ansible/roles/raspi-config/tasks/main.yml

```diff
@@ -33,4 +33,11 @@
   command: raspi-config nonint do_i2c 1
 
 - name: Enable camera
-  command: raspi-config nonint do_camera 1  
\ No newline at end of file
+  command: raspi-config nonint do_camera 1  
+
+- name: modifie the dhcpd5 service
+  template:
+    src: dhcpd5.service.j2
+    dest: /etc/systemd/system/dhcpcd5.service
```

Comment:
I think it's the wrong file name. You should then notify and have an handler to reload the deamon and ensure the service is enabled (it would be cleaner).

Correct path is `/lib/systemd/system/dhcpcd.service`